packages/flutter/lib/src/gestures/velocity_tracker.dart

```diff
@@ -208,6 +208,34 @@ class _LeastSquaresVelocityTrackerStrategy extends _VelocityTrackerStrategy {
 
 }
 
+/// A velocity in two dimensions.
+class Velocity {
+  const Velocity({ this.pixelsPerSecond });
+
+  /// A velocity that isn't moving at all.
+  static const Velocity zero = const Velocity(pixelsPerSecond: Offset.zero);
+
+  /// The number of pixels per second of velocity in the x and y directions.
+  final Offset pixelsPerSecond;
+
+  /// The square of the magnitude of the velocity (in pixels per second) squared.
+  double get speedSquared {
+    return pixelsPerSecond.dx * pixelsPerSecond.dx
```

Comment:
I just removed it and made the one caller do that, which has the advantage of putting the units in the call site.